src/lib/statistics/abstract_cardinality_estimator.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <memory>
+
+#include "cardinality_estimation_cache.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+class AbstractLQPNode;
+class TableStatistics;
+
+/**
+ * Base class for algorithms determining the output cardinality/statistics of an LQP during optimization.
+ */
+class AbstractCardinalityEstimator {
+ public:
+  virtual ~AbstractCardinalityEstimator() = default;
+
+  /**
+   * @return a new instance of this estimator with mint caches. Used so that caching guarantees can be enabled on the
```

Comment:
Not sure how commonly understood "mint" is here. "Empty" would serve the same purpose.

Reply:
changed to "empty"